Doc/Manual/Guile.html

```diff
@@ -454,14 +453,13 @@
 Object ownership and %newobject</a> in the SWIG manual.  All typemaps use an $owner var, and
 the guile module replaces $owner with 0 or 1 depending on feature:new.</p>
 
-<H2><a name="Guile_nn12">24.8 Native Guile pointers</a></H2>
-
+<H2><a name="Guile_nn12">24.7 Native Guile pointers</a></H2>
```

Comment:
Yeah... I saw that and the other section issues you noted. The documentation stuff (indices, section numbers) is maintained partially by scripts. I just figured it was generated elsewhere.

But of course, now that I think about it, that wouldn't really explain why the section numbers are up-to-date in the repository.

I regenerated the section numbers and everything and committed them.

Sorry about that.